app/models/manageiq/providers/redhat/infra_manager/vm_import.rb

```diff
@@ -43,6 +56,13 @@ def validate_import_vm
     Gem::Version.new(api_version) >= Gem::Version.new('4')
   end
 
+  def submit_configure_imported_vm_networks(userid, vm_id)
```

Comment:
@gmcculloug than we have to come up with different name for `configure_imported_vm_networks` https://github.com/ManageIQ/manageiq-providers-ovirt/pull/47/files#diff-9093cd3391ba88c199808e385f9304e0R35 since the `submit_` method actually only enqueues the actual `configure_imported_vm_networks` method.